### PR TITLE
docs(new pipeline): Added highlight article for new pipeline and two relevant diagrams

### PR DESCRIPTION
**Pull request by barieom**

### CODE REVIEW

File: website/content/en/highlights/2021-11-16-new-pipeline.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+To illustrate this better, let's assume we have a pipeline with three stages - source, transforms, then sink. The `transforms` directory has the following structure:
+
+```
+/transforms/
+└───processing/
+│   └───logs/   
+│       │   sudo.toml
+│       │   aws_s3_access_logs.toml
+│
```
Comment (jdrouet) [inline website/content/en/highlights/2021-11-16-new-pipeline.md:33]: expanding this in subfolders is not yet implemented 😉

**Review by jdrouet — CHANGES_REQUESTED** [review]

File: website/content/en/highlights/2021-11-16-new-pipeline.md (hunk 1 above)
Comment (jszwedko) [reply website/content/en/highlights/2021-11-16-new-pipeline.md:33]: 👍 Do you mean to say that users can only define pipelines as top-level configuration?

If so, I might be inclined to leave this highlight for 0.19.0 once we've implemented the subdirectory loading.

Comment (barieom) [reply website/content/en/highlights/2021-11-16-new-pipeline.md:33]: @binarylogic - what're your thoughts?

Comment (binarylogic) [reply website/content/en/highlights/2021-11-16-new-pipeline.md:33]: Agree, I would like to announce this once we support pipelines-per-file.

Comment (jdrouet) [reply website/content/en/highlights/2021-11-16-new-pipeline.md:33]: > Do you mean to say that users can only define pipelines as top-level configuration?

Yes, only at the top level or in the `transforms` folder, but for now, you cannot split it more.

File: website/content/en/highlights/2021-11-16-new-pipeline.md [hunk 2]
```diff
@@ -114,7 +112,7 @@ type = "pipelines"
 inputs = ["datadog_agent", "syslog"]
 mode = "linear"
 
-logs.order = [
+.order = [
     "aws_s3_access_logs",
     "sudo"
 ]
```
Comment (jdrouet) [inline website/content/en/highlights/2021-11-16-new-pipeline.md]:
```suggestion
order = [
    "aws_s3_access_logs",
    "sudo"
]
```

File: website/content/en/highlights/2021-11-16-new-pipeline.md [hunk 3]
```diff
@@ -20,16 +19,15 @@ Vector's topology is based on a [directed acylclic graph], enabling users to cre
...
 ```
 /transforms/
 └───processing/
-│   └───logs/   
-│       │   sudo.toml
-│       │   aws_s3_access_logs.toml
+│    │   sudo.toml
+│    │   aws_s3_access_logs.toml
 │   
 │   processing.toml
```
Comment (jdrouet) [inline website/content/en/highlights/2021-11-16-new-pipeline.md]: Sorry, event the `processing` folder is not implemented yet. I'm working on it.
For now, you can only have the `processing.toml`

File: website/content/en/highlights/2021-11-16-new-pipeline.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
+---
+date: "2021-11-16"
+title: "Pipelines is now available on Vector"
+description: "New pipelines now enables further collaboration"
+authors: ["barieom"]
+pr_numbers: []
+release: "0.18.0"
```
Comment (jszwedko) [inline website/content/en/highlights/2021-11-16-new-pipeline.md:7]:
```suggestion
release: "0.19.0"
```

File: website/content/en/highlights/2021-11-16-new-pipeline.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+And in the processing.toml file, the following configuration is required to set the order of the processors:
+
+``` toml
+type = "pipelines"
+inputs = ["datadog_agent", "syslog"]
+mode = "linear"
+
+.order = [
```
Comment (jszwedko) [inline website/content/en/highlights/2021-11-16-new-pipeline.md:115]:
```suggestion
order = [
```

File: website/content/en/highlights/2021-11-16-new-pipeline.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+    "aws_s3_access_logs",
+    "sudo"
+]
+```
+
+
+For our next steps, we'll be looking to add `filtering` functionality, which will enable events to be passed to the next processor if a condition is not. In the meantime, if you any feedback for us, let us know on our [Discord chat][] or on [Twitter][]!
+
```
Comment (jszwedko) [inline website/content/en/highlights/2021-11-16-new-pipeline.md:123]: Filtering is used above, no?

**Review by jszwedko — COMMENTED** [review]
🎉 I think this one could be brushed off for 0.19.0.

**Comment by jszwedko** [comment]
I think this highlight can go out in 0.20.0.

**Comment by jszwedko** [comment]
Closing this since this transform has been deprioritized for the foreseeable future.

**Pull request closed without merging**